test/sanity/validate-modules/ignore.txt

```diff
@@ -99,8 +99,7 @@ lib/ansible/modules/cloud/centurylink/clc_server_snapshot.py E335
 lib/ansible/modules/cloud/cloudstack/cs_cluster.py E326
 lib/ansible/modules/cloud/cloudstack/cs_host.py E326
 lib/ansible/modules/cloud/cloudstack/cs_iso.py E323
-lib/ansible/modules/cloud/cloudstack/cs_loadbalancer_rule.py E322
-lib/ansible/modules/cloud/cloudstack/cs_loadbalancer_rule.py E324
+lib/ansible/modules/cloud/cloudstack/cs_loadbalancer_rule.py E309
```

Comment:
Please remove this line

Reply:
@gundalow Done

Reply:
@gundalow As `validate-module` seems to do its diff against the devel branch, couldn't we keep ignoring the error for this PR and then fix it by another PR when it will be merged?